webpack/webpack.dev.js

```diff
@@ -73,6 +74,17 @@ const devConfig = {
     ],
     module: {
         rules: [
+            {
+                test: /\.s?css$/,
+                include: [paths.component],
+                use: ['style-loader', typingsForCssModulesLoader(), postCSSLoader, 'sass-loader']
+            },
+            {
```

Comment:
treating global styles (`global.scss` at the root of the `/src` directory) as a global style sheet.